### PR TITLE
[SP-49] Created validation for Change Password form

### PR DESCRIPTION
**Pull request by sanya-terror**

### CODE REVIEW

File: lib/account/change-password/change-password.component.dart [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+
+   ChangePasswordComponent() {
+
+      this.form = new ControlGroup({
+         'password': new Control('', Validators.compose([PasswordValidator.validate, Validators.required])),
+         'passwordRepeat': new Control('', Validators.compose([PasswordValidator.validate, Validators.required])),
+      });
+
+      this.messages = {
+         'password': 'Password is required',
```
Comment (Ne4istb) [inline lib/account/change-password/change-password.component.dart]: password and passwordRepeat are the same so you can have only one message

Comment (sanya-terror) [inline lib/account/change-password/change-password.component.dart]: @Ne4istb , yes of course, but I've thought about further implementation, we have to create dynamic messages, for instance passwordRepeat will has such kinds of messages as:
"Passwords aren't mutched" or somethink like that. So what do you think so about it? If you still have some objections, I'll drop it, if you aren't => merge it)))

Comment (sanya-terror) [inline lib/account/change-password/change-password.component.dart]: @Ne4istb anyway, I think all messages ought to be dynamic, as "Too short length" or "Too long length", or "Incorrect input value" or smth like that. I think it make sence, isn't it?

Comment (Ne4istb) [inline lib/account/change-password/change-password.component.dart]: np, i just trying to follow KISS principle. Anyway, it's temporal solution which will be replaced when we will implement localization mechanism

**Pull request merged**